powerbi-docs/visuals/power-bi-visualization-export-data.md

```diff
@@ -149,24 +149,26 @@ When you select **Export**, Power BI exports the data to an *.xlsx* file and you
 
     - Since we applied filters to the visualization, the exported data will export as filtered. Notice that the first row displays **Applied filters: City is not Atlanta, GA**. 
 
-## Protecting proprietary data
+## Customize export data user expirience
 
-Your dataset may have content that should not be seen by all users. If you are not careful, exporting underlying data may let users see all the detailed data for that visual -- every column and every row in the data. 
+Users that granted access to report are **granted access to the entire dataset** unless [row-level security (RLS)](../admin/service-admin-rls.md) limits their access. Report authors and Power BI adminstrators may use capabilities below to customize user experiences when viewing or interacting or export data from reports: 
```

Comment:
Users who are granted access to a report are **granted access to the entire underlying dataset**, unless [row-level security (RLS)(../admin/service-admin-rls.md) limits their access. Report authors and Power BI administrators can use the capabilities described below to customize the user experience.